.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "files.exclude": {
+    "**/.git": true,
+    "**/.svn": true,
+    "**/.hg": true,
+    "**/CVS": true,
+    "**/.DS_Store": true
+  },
+  "search.exclude": {
+    "**/dist": true,
+    "**/bin": true
+  },
+  "editor.tabSize": 4,
+  "editor.formatOnSave": true,
+  "rewrap.wrappingColumn": 120
```

Comment:
Are these something that we want to enforce in eslint?

Reply:
I decided to leave things as generated after following steps [here](https://github.com/salesforcecli/cli/wiki/Code-Your-Plugin#generate-stuff), which means that some of these settings are not placed in eslint rules.

In my experience, if we move these to eslint, then some of them interfere with other styling rules (specially with prettier). For example if I add the eslint rule `'indent': ['error', 4]` then there will be cases where eslint will generate error b/c it thinks a line should start with 8 spaces but prettier wants the line to start with 12 spaces (I think Prettier style should actually be the one that wins b/c it results in a nicer style). Same issue goes with `'quotes': ['error', 'single'],` eslint rule where eslint would want to enforce single quote all the time but prettier would prefer you to use double quotes some time, so the two would clash.

As such I think we should just leave the rules as they are right now